router/models/auth/auth.go

```diff
@@ -0,0 +1,16 @@
+package auth
+
+import (
+	userModel "github.com/TheDevExperiment/server/internal/db/users/models"
+)
+
+type AuthResponse struct {
+	Message string           `form:"message" json:"message" xml:"message"  binding:"required"`
+	Data    []userModel.User `form:"data" json:"data"`
```

Comment:
I dont think multiple users can be returned
```suggestion
	Data    userModel.User `form:"data" json:"data"`
```